tests/unit/commando/core_test.py

```diff
@@ -1,6 +1,23 @@
-from commando.core import hello_world
+from unittest.mock import sentinel
 
+import pytest
 
-class TestHelloWorld:
+from commando.core import commando
+
+
+class TestCommando:
     def test_it(self):
-        assert hello_world() == "Hello, world!"
+        exit_status = commando(
+            [sentinel.repo_1, sentinel.repo_2],
+            ["test", "command"],
+            sentinel.commit_message,
+            sentinel.branch,
+            sentinel.pr_title,
+            sentinel.pr_body,
+        )
+
+        assert not exit_status
```

Comment:
I could write proper unit tests for the logic in the `commando()` function. Not totally sure that it's worth the bother though